tests/introspection/ApiSelectorTest.cs

```diff
@@ -93,6 +93,12 @@ protected virtual bool Skip (Type type, string selectorName)
 					return true;
 				}
 				break;
+			case "AVContentKeyRequest":
+				switch (selectorName) {
+				case "renewsExpiringResponseData":
```

Comment:
This comes from a category `AVContentKeyRequestRenewal (AVContentKeyRequest)` and it might be better to bind it as such. The problem with ignoring (in tests) is that it's likely to throw a native exception when executed, which is what the managed debugger will do when it shows any instance (in XS or VS) of this type - stopping the app execution (bad).